Cargo.toml

```diff
@@ -94,7 +99,7 @@ zeroize_crate = { package = "zeroize", version = "1", default-features = false,
 
 [dev-dependencies]
 hex = "0.4.2"
-page_size = "0.5.0"
+page_size = "0.6.0"
```

Comment:
I just pushed d6265dafc9366d59de273a831339dafe0058ceb7 for this, so it should rebase out.